keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -277,18 +276,18 @@ class ApplicationService(
     }
 
     val artifactSummaries = deliveryConfig.artifacts.map { artifact ->
+      // We calculate a bunch of data for each artifact, so pre-load the bulk of it here and pass that info into
+      //  later functions to save redundant database calls.
```

Comment:
```suggestion
      // later functions to save redundant database calls.
```

Reply:
I purposefully put spaces on the front of continuation comments so they're visually grouped